hsa-amd-aqlprofile/PKGBUILD

```diff
@@ -1,30 +1,23 @@
 # Maintainer: Ilya Elenskiy <elenskiy.ilya@gmail.com>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
 pkgname=hsa-amd-aqlprofile
 pkgver=1.0.0
```

Comment:
I think it is fine to keep it as 1.0.0 since that is what the debian package is at.